enforceBeachball.js

```diff
@@ -0,0 +1,10 @@
+const cp = require('child_process');
+
+cp.exec('yarn changefile', { timeout: 30000 }, (err, stdout, stderr) => {
+  if (!err && stdout.includes('No change files are needed')) {
+    console.log('Beachball guidelines were correctly followed. Continuing...');
+    return;
+  } else {
+    throw new Error("Change files are required before merging. Please run 'yarn changefile' from the monorepo root.");
```

Comment:
>"Change files are required before merging. Please run 'yarn changefile' from the monorepo root." [](http://example.com/codeflow?start=20&length=96)

I don't know how fancy you want to make this but if someone sees this error, is it worth not just saying "run this" but also possibly "see this: [https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md#change-log-using-beachball](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md#change-log-using-beachball)" ?